conftest.py

```diff
@@ -0,0 +1,33 @@
+"""PyTest Fixtures."""
+import os
+
+import pytest
+
+try:
+    from ansiblelint.constants import DEFAULT_RULESDIR
+    from ansiblelint.rules import RulesCollection
+    from ansiblelint.testing import RunFromText
+except ImportError:
+    pytest.exit("You need to install ansiblelint to be able to load test fixtures.")
```

Comment:
I don't think this is necessary.

Reply:
It is highly desirable. Try to just run "pytest" in repository folder without it and it will fail in an ugly way without it. I prefer to give a hint to the developer instead of risking more tickets, especially because installing the library before running tests is not something most project require, many of them work without that.

Reply:
Supporting pytest outside of tox is wrong.